Ginger/GingerCoreCommon/Actions/ActOutDataSourceConfig.cs

```diff
@@ -48,42 +49,98 @@ public  static partial class Fields
 
         [IsSerializedForLocalRepository]
         public string DSTable { get; set; }
-        
-        [IsSerializedForLocalRepository]
-        public bool Active { get { return mActive; } set { mActive = value; OnPropertyChanged(Fields.Active); } }
 
-        [IsSerializedForLocalRepository]
-        public string OutputType { get; set; }
+        //[IsSerializedForLocalRepository]
+        //public bool Active { get { return mActive; } set { mActive = value; OnPropertyChanged(Fields.Active); } }
+
+        //[IsSerializedForLocalRepository]
+        //public string OutputType { get; set; }
 
         [IsSerializedForLocalRepository]
         public string OutParamMap { get; set; }
-        
+
         public override string ItemName
         {
             get
             {
-                return this.OutputType;
+                return this.DSName;
             }
             set
             {
-                this.OutputType = value;
+                this.DSName = value;
             }
         }
 
-        ObservableList<string> mPossibleValues = new ObservableList<string>();
-        public ObservableList<string> PossibleValues
+        //ObservableList<string> mPossibleValues = new ObservableList<string>();
+        //public ObservableList<string> PossibleValues
+        //{
+        //    get
+        //    {
+        //        return mPossibleValues;
+        //    }
+        //    set
+        //    {
+        //        mPossibleValues = value;
+        //        OnPropertyChanged(Fields.PossibleValues);
+        //    }
+        //}
+
+        //string mTableColumn;
+        //[IsSerializedForLocalRepository]
+        //public string TableColumn
+        //{
+        //    get
+        //    {
+        //        return mTableColumn;
+        //    }
+        //    set
+        //    {
+        //        mTableColumn = value;
+        //        OnPropertyChanged(Fields.TableColumn);
+        //    }
+        //}
+
+        [IsSerializedForLocalRepository]
+        public ObservableList<ActOutDataSourceConfigParam> ActOutDataSourceConfigParameters  = new ObservableList<ActOutDataSourceConfigParam>();
+
+
+        public override bool SerializationError(SerializationErrorType errorType, string name, string value)
         {
-            get
+            if (errorType == SerializationErrorType.PropertyNotFound)
             {
-                return mPossibleValues;
-            }
-            set
-            {
-                mPossibleValues = value;
-                OnPropertyChanged(Fields.PossibleValues);
+                if (name == "Active" || name == "OutputType" || name == "TableColumn")
+                {
+                    return true;
+                }
             }
+            return false;
+        }
+
+        public override void PostSerialization()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a nested comment explaining why this method is empty, throw a 'NotSupportedException' or complete the implementation.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4815237)